great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -449,6 +440,128 @@ class MetricDomainTypes.
                     "SqlAlchemyExecutionEngine only supports the great_expectations condition_parser."
                 )
 
+        if "column" in domain_kwargs:
+            return selectable
+
+        if "column_A" in domain_kwargs and "column_B" in domain_kwargs:
+            if "ignore_row_if" in domain_kwargs:
+                if self.active_batch_data.use_quoted_name:
+                    # If case matters...
+                    # noinspection PyPep8Naming
+                    column_A_name = quoted_name(domain_kwargs["column_A"])
+                    # noinspection PyPep8Naming
+                    column_B_name = quoted_name(domain_kwargs["column_B"])
+                else:
+                    # noinspection PyPep8Naming
+                    column_A_name = domain_kwargs["column_A"]
+                    # noinspection PyPep8Naming
+                    column_B_name = domain_kwargs["column_B"]
+
+                ignore_row_if = domain_kwargs["ignore_row_if"]
+                if ignore_row_if == "both_values_are_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.and_(
```

Comment:
you're reusing the column name in both places...which makes me think we must not have a test exercising this logic yet. Can you ensure we have a test in place to validate these cases?

Reply:
@jcampbell Great catch!  This was an unfortunate typo.  I corrected it and added a test case to exercise the functionality.